### PR TITLE
Add circleci integration

### PR DESCRIPTION
**Pull request by peter7z**
Adds circleci to run tests.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+            bundle exec rspec \
+              --format progress \
+              $TEST_FILES
+
+      # collect reports
+      - store_test_results:
+          path: /tmp/test-results
+      - store_artifacts:
+          path: /tmp/test-results
+          destination: test-results
```
Comment (TimoPeraza) [inline .circleci/config.yml]: missing newline 😆

File: .rubocop.yml [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  Enabled: false
+
+Style/FrozenStringLiteralComment:
+  Enabled: false
+
+Style/ModuleFunction:
+  Enabled: false
+
+Style/ReturnNil:
+  Enabled: true
```
Comment (TimoPeraza) [inline .rubocop.yml]: missing newline 😆

File: .circleci/database.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+test:
+  default: &default
+  adapter: postgresql
+  encoding: unicode
+  pool: 5
+  username: postgres
+  password: postgres
+  host: localhost
+  port: 5432
+  database: test
```
Comment (TimoPeraza) [inline .circleci/database.yml]: missing newline 😆

**Comment by TimoPeraza** [comment]
doesn't circleci need application.yml file? 🤔

**Comment by peter7z** [comment]
Not necessarily @TimoPeraza

**Pull request merged**